### PR TITLE
ceph-ansible: add add_mgrs scenario to CI

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Add add_mgrs scenario to CI so that it can be tested against master and
nautilus.

### CODE REVIEW

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml [hunk 1]
```diff
@@ -293,6 +293,48 @@
...
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            set -x
+            # if the target branch is master we RUN these tests.
+            if [[ "$ghprbTargetBranch" =~ "stable-" ]]; then
+              exit 1
+            fi
+            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'infrastructure-playbooks/add-mgrs'
```
Comment (dsavineau) [inline ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: probably need to add '|tests/functional/add-mgrs' there (like add-mons)

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml [hunk 2]
```diff
@@ -293,6 +293,48 @@
...
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            set -x
+            # if the target branch is stable-4.0 we RUN these tests.
+            if [[ "$ghprbTargetBranch" =~ stable-3.[0-2]|master ]]; then
+              exit 1
+            fi
+            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'infrastructure-playbooks/add-mgrs'
```
Comment (dsavineau) [inline ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: ditto

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml (hunk 1 above)
Comment (rishabh-d-dave) [reply ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: Oops. I was supposed to keep "tests/functional/add-mgrs" and was supposed to remove "infrastructure-playbooks/add-mgrs". I've fixed it.

File: ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml (hunk 2 above)
Comment (rishabh-d-dave) [reply ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml]: Done.

**Pull request merged**